pkg/test/resourcefixture/testdata/basic/bigqueryconnection/v1alpha1/bigqueryconnectionconnection/awsconnectionbasic/_http.log

```diff
@@ -66,7 +66,7 @@ X-Xss-Protection: 0
 
 ---
 
-PATCH https://bigqueryconnection.googleapis.com/v1/projects/${projectId}/locations/aws-us-east-1/connections/71389360-831c-431d-8975-837aee2153be?%24alt=json%3Benum-encoding%3Dint&updateMask=friendlyName%2Cdescription
```

Comment:
Not sure why "name" is included in the updateMask?

Reply:
Good catch. Looks like the auto mapper does not map `name` between proto and krm (I think that's right because this is a service-generated ID). I updated the code and golden log.